package.json

```
@@ -30,7 +30,7 @@
     "migration:revert": "typeorm-ts-node-esm -d ./src/config/typeorm.datasource.ts migration:revert",
     "seed:run": "typeorm-ts-node-esm -d ./src/config/typeorm-seeds.datasource.ts migration:run",
     "seed:revert": "typeorm-ts-node-esm -d ./src/config/typeorm-seeds.datasource.ts migration:revert",
-    "firebase:token": "node ./scripts/get-token.js",
+    "firebase:token": "node --experimental-loader ./scripts/get-token.js",
```

```suggestion
    "firebase:token": "node ./scripts/get-token.js",
```
I think this works without this option on both Node 16 and Node 18.
It produces extra output which is not easily silenced and it prevents me from using it programmatically with my httpie flow with:
```shell
export JWT_AUTH_TOKEN=$(npm run --silent firebase:token)`
```